File: web/lib/src/chat/models/flight.dart
import 'dart:convert';

class Flight {

  String origin;
  String destination;
  String departDate;
  String gate;
  double price;

  Flight({this.destination, this.origin, this.gate, this.departDate, this.price});

  factory Flight.fromJson(Map<String, dynamic> data){
    return Flight(
      departDate: data['depart_date'],
      destination: data['destination'],
      origin: data['origin'],
      price: data['price'].toDouble(),
      gate: data['gate']
    );
  }
}

File: back/models/flight.py
class Flight:
    def __init__(self, data):
        self.destination = data.get('destination')
        self.origin = data.get('origin')
        self.depart_date = data.get('depart_date')
        #self.depart_date = data.get('departure_date')
        self.price = data.get('value')
        self.gate = data.get('gate')
        #self.price = data.get('price')
        #self.currency = data.get('currency')

